rego/Drive.rego

```diff
@@ -152,13 +239,45 @@ NoSuchEvent1_4(TopLevelOU) := true if {
 
 default NoSuchEvent1_4(_) := false
 
-NonCompliantOUs1_4 contains OU if {
+GetFriendlyValue1_4(Value_A, Value_B, AcceptableValues_A, AcceptableValues_B) :=
+"External Sharing is Disabled" if {
+    Value_B in AcceptableValues_B
+} else := concat("", ["External Sharing is Enabled, ",
+    "but Sharing invites to non-google accounts is disabled"]) if {
```

Comment:
Another minor grammatical tweak. Also "items" are what are being shared, not invites.
```suggestion
    "but sharing items to non-google accounts is disabled"]) if {
```